inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/model/db-changelog.xml

```diff
@@ -331,4 +331,35 @@
       </column>
     </addColumn>
   </changeSet>
+
+  <changeSet author="INCEpTION Team" id="20180503">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <columnExists tableName="knowledgebase" columnName="descriptionIri"/>
+      </not>
+    </preConditions>
+
+    <addColumn tableName="knowledgebase">
+      <column name="descriptionIri"
+              type="VARCHAR(255)"
+              defaultValue="http://www.w3.org/2000/01/rdf-schema#comment">
+        <constraints nullable="false" />
+      </column>
+    </addColumn>
+  </changeSet>
+
+  <changeSet author="INCEpTION Team" id="20180511">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <columnExists tableName="knowledgebase" columnName="ftsIri"/>
+      </not>
+    </preConditions>
+
+    <addColumn tableName="knowledgebase">
+      <column name="ftsIri"
+              type="VARCHAR(255)">
```

Comment:
That should be non-null, right? Or can it be null in which case full-text search (i.e. concept linking) would not be supported and we could remove the "supports concept linking" checkbox?